### ISSUE TITLE
Building docker image: "gpg: can't open '/tmp/mosq.tar.gz.asc': No such file or directory"

### ISSUE THREAD

**Issue by schildbach** [eclipse/mosquitto#1944]
On Ubuntu 20.04 with `Docker version 19.03.8, build afacb8b7f0`,
using this git repo from tag `v2.0.2`,
running `docker build .` from the `docker/2.0` or `docker/2.0-openssl` directories,
yields this error:

```
+ wget https://github.com/warmcat/libwebsockets/archive/v2.4.2.tar.gz -O /tmp/lws.tar.gz
wget: bad address 'github.com'
Fetching GPG key A0D6EEA1DCAE49A635A3B2F0779B22DFB3E717B7 from ha.pool.sks-keyservers.net
+ echo 'Fetching GPG key A0D6EEA1DCAE49A635A3B2F0779B22DFB3E717B7 from ha.pool.sks-keyservers.net'
+ gpg --keyserver ha.pool.sks-keyservers.net --keyserver-options 'timeout=10' --recv-keys A0D6EEA1DCAE49A635A3B2F0779B22DFB3E717B7
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: /root/.gnupg/trustdb.gpg: trustdb created
gpg: key 779B22DFB3E717B7: public key "Roger A. Light <roger@atchoo.org>" imported
gpg: Total number processed: 1
gpg:               imported: 1
+ found=yes
+ break
+ test -z yes
+ gpg --batch --verify /tmp/mosq.tar.gz.asc /tmp/mosq.tar.gz
gpg: can't open '/tmp/mosq.tar.gz.asc': No such file or directory
gpg: verify signatures failed: No such file or directory
```

**Comment by ralight** [comment]
`wget: bad address 'github.com'`

It looks like you have DNS problems.

**Comment by schildbach** [comment]
I saw that too, but outside of docker the `wget https://github.com/warmcat/libwebsockets/archive/v2.4.2.tar.gz` line works fine.

**Comment by schildbach** [comment]
I resolved this issue by adding `"dns": ["8.8.8.8"]` to `/etc/docker/daemon.json`. I've no idea why only (!) github.com cannot be resolved on my provider's DNS server only (!) within Docker.
